geoportailv3/static/js/layerinfo/layerinfodirective.js

```diff
@@ -0,0 +1,51 @@
+goog.provide('app.layerinfoDirective');
+
+goog.require('app');
+goog.require('app.ShowLayerinfo');
+
+
+/**
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+app.layerinfoDirective = function(appLayerinfoTemplateUrl) {
+  return {
+    restrict: 'E',
+    scope: {
+      'layer': '=appLayer'
```

Comment:
Our convention is to repeat the name of the directive, so `appLayerinfoLayer` instead of `appLayer`. I know it's more verbose, but I'd favor consistency.